azure-devops/azext_devops/dev/pipelines/pipeline.py

```diff
@@ -96,8 +100,25 @@ def pipeline_show(id=None, name=None, open=False, organization=None, project=Non
     return build_definition
 
 
+def set_param_variable(variables, as_dict=False, argument='variables'):
+    param_variables = None
+    if variables is not None and variables:
+        param_variables = {}
+        for variable in variables:
+            separator_pos = variable.find('=')
+            if separator_pos >= 0:
+                if as_dict:
+                    param_variables[variable[:separator_pos]] = {"value": variable[separator_pos + 1:]}
+                else:
+                    param_variables[variable[:separator_pos]] = variable[separator_pos + 1:]
+            else:
+                raise ValueError(
+                    f'The --{argument} argument should consist of space separated "name=value" pairs.')
```

Comment:
can you share the screenshot on how this will look to customer ?

Reply:
If it is going to look just like 
```
The --{argument} argument should consist of space separated "name=value" pairs.
```
then how would it help customers ?

Reply:
![image](https://user-images.githubusercontent.com/83656375/141441063-7618779c-5f09-4613-afd5-b1b9f739a457.png)